public/app/features/explore/ExploreActions.tsx

```diff
@@ -39,7 +39,7 @@ export const ExploreActions = () => {
         },
         section: exploreSection,
       });
-      if ([panes[1]]) {
+      if (panes[1]) {
```

Comment:
@Elfo404 same question here: does this match what the intention was for this bit of code? 🤔

Reply:
Most likely yes, it must have been a typo we missed